src/app/project/CreateEditFeature.js

```diff
@@ -121,12 +121,24 @@ define([
         //      sent to toaster
         duplicateActionMsg: 'Can\'t add duplicate actions!',
 
+
         // Properties to be sent into constructor
 
+        // existingData: Object (optional)
+        // {
+        //     category: String,
+        //     retreatment: Boolean,
+        //     geometry: Geometry,
```

Comment:
I think that's a possibility for sure. Why not just assume that everything coming in is new and recreate it all?